src/components/AssignMemberToTeamButton/index.js

```diff
@@ -0,0 +1,266 @@
+import React, { Component, Fragment } from 'react';
+import { Modal, Form, Loader } from 'semantic-ui-react';
+import gql from 'graphql-tag';
+import { Query, Mutation, withApollo } from 'react-apollo';
+
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import SelectInput from 'components/SelectInput';
+import ErrorMessage from 'components/ErrorMessage';
+import { StyledButton } from '../../styles/Group';
+
+import { ALL_USERS_QUERY } from '../ProjectFormCreate';
+
+class AssignMemberToTeamButton extends Component {
+  state = {
+    open: false,
+    membersToBeUpdated: []
+  };
+
+  getOptions = (allUsers, teamUsers) => {
+    // this function returns users who is not belongingto given teamUsers
+    const teamUsersId = teamUsers.map(user => user.id);
+    const filteredUsers = allUsers.filter(user => !teamUsersId.includes(user.id));
```

Comment:
you can use reduce instead of filter and map
options = allUsers.reducer((result, user) => {
  if(!teamUsersId.includes(user.id)) {
     return result.concat({
      value: user.id,
      label: user.name
    })
  }
  return result
})

Reply:
@gianglevan94 
This is smart. I will fix 🙏